File: src/main/java/net/xboxman/andonium/util/Registers/Registers.java
package net.xboxman.andonium.util.Registers;

import net.neoforged.bus.api.IEventBus;
import net.xboxman.andonium.block.ModBlocks;
import net.xboxman.andonium.item.KotlinModItems;
import net.xboxman.andonium.item.ModCreativeModeTabs;
import net.xboxman.andonium.item.ModItems;
import net.xboxman.andonium.util.ModMenus;

public class Registers {

    public static void registerAll(IEventBus eventBus) {
        ModItems.register(eventBus);
        KotlinModItems.INSTANCE.register(eventBus);
        ModBlocks.register(eventBus);
        ModCreativeModeTabs.register(eventBus);
        ModMenus.REGISTRY.register(eventBus);
    }
}

File: src/main/java/io/github/coolman4567/andonium/kotlin/util/registers/Registers.kt
package io.github.coolman4567.andonium.kotlin.util.registers

import io.github.coolman4567.andonium.block.ModBlocks
import io.github.coolman4567.andonium.item.ModCreativeModeTabs
import io.github.coolman4567.andonium.item.ModItems
import io.github.coolman4567.andonium.kotlin.block.KotlinModBlocks
import io.github.coolman4567.andonium.kotlin.item.KotlinModItems
import io.github.coolman4567.andonium.util.ModMenus
import net.neoforged.bus.api.IEventBus

object Registers {
    @JvmStatic
    fun registerAll(eventBus: IEventBus) {
        ModItems.register(eventBus)
        KotlinModItems.register(eventBus)
        KotlinModBlocks.register(eventBus)
        ModBlocks.register(eventBus)
        ModCreativeModeTabs.register(eventBus)
        ModMenus.REGISTRY.register(eventBus)
    }
}
